src/main/java/org/catmaid/ScaleCATMAID.java

```diff
@@ -72,46 +72,55 @@
 	{
 		public int tileWidth;
 		public int tileHeight;
+		public long minC;
```

Comment:
IMO we need all four parameters considering that the retiler can take all four params - I don't think it hurts having them anyway. If you prefer them to be in pixel coordinates at scale 0 that's perfectly fine and it is consistent with the retiler.